### DISCUSSION TITLE
Looking for feedback on evaluation-ids.xsd

### DISCUSSION THREAD

**Discussion by vanderpol** [OVAL-Community/OVAL#209] [General]
Does anyone have any knowledge on **evaluation-ids.xsd**?  To me it seems like an artifact from the old MITRE tool 'ovaldi', when testing my OVAL 6.0 updates, I see that it fails basic XML validation in a few tools.  I think it can be just deleted, but wanted to pass it by others.

@solind, @A-Biggs, @wmunyan 

using 'lxml'

```
Unexpected error while validating 'evaluation-ids.xsd': element decl. '{http://oval.mitre.org/XMLSchema/ovaldi/evalids}definition', attribute 'type': The QName value '{http://oval.mitre.org/XMLSchema/oval-common-5}DefinitionIDPattern' does not resolve to a(n) type definition., line 21
```

using 'xmlschema'

```
Failed to load XSD file evaluation-ids.xsd: unknown type 'oval:DefinitionIDPattern':

Schema component:

  <xsd:element xmlns:xsd="http://www.w3.org/2001/XMLSchema" name="definition" type="oval:DefinitionIDPattern" minOccurs="1" maxOccurs="unbounded">
      <xsd:annotation>
          <xsd:documentation>Each definition represents the id of a definition to be evaluated.</xsd:documentation>
      </xsd:annotation>
  </xsd:element>

Path: /xsd:schema/xsd:element/xsd:complexType/xsd:sequence/xsd:element

Schema URL: file:///evaluation-ids.xsd

**Comment by solind** [comment]
It looks like the XML definitions schema wasn't loaded for some reason.  DefinitionIDPattern is definitely found there.  It's probably just a matter of a failure to import it from a document location (since it's not available to be downloaded from the namespace URL).

Anyway, it's an XML document format that was accepted by Sovaldi, for listing the IDs of definitions you want to evaluate in the context of a specific OVAL definitions document (containing multiple definitions).  What has always bugged me the most about it is the spelling error in the top-level XML element `evaluatation-ids` (should be `evaluation-ids`).

I think it's probably safe to get rid of it.
